lib/oban/worker.ex

```diff
@@ -254,6 +269,78 @@ defmodule Oban.Worker do
           if MyApp.something?(job), do: :ok, else: {:snooze, 60}
         end
       end
+
+  ## Workers in A Different Application
+
+  Occasionally you may need to insert a job for a worker that exists in another
+  application. In that case you can use `Oban.Job.new/2` to build the changeset
+  manually:
+
+      %{id: 1, user_id: 2}
+      |> Oban.Job.new(queue: :default, worker: OtherApp.Worker)
+      |> Oban.insert()
+
+  ## Scheduling Jobs
```

Comment:
I think scheduling jobs and prioritizing jobs belong here; thoughts?

Reply:
Scheduling might not make as much sense to be here. Maybe it does? Do you feel it gets lost here? Prioritizing belongs here, because it's an option you define on the worker.

Reply:
Yeah maybe scheduling is not the same... But I don't know, a page on its own? Maybe it makes sense. Let's go with that for now

Reply:
Done, moved to its own guide.